### DISCUSSION TITLE
Exclude folders

### DISCUSSION THREAD

**Discussion by HadesNinth** [logancyang/obsidian-copilot#553] [Q&A]
I sometimes use the feature "Save as Note." The AI would sometimes pull information from these notes for its QA answers, which is quite frankly a lower quality source than the actual note themselves.

So, I'd like a setting to set a couple folders to be excluded, so that I could exclude "copilot-conversations" from being indexed by the AI (unless explicitly mentioned via [[note]]).

This would also help with the AI having to index my "Templates" folder, which just stores a bunch of templates for Templater, and my private Journal folder.

Anyways, I extend my thanks to you, Logan, and all others that have worked on this plugin. It changed the way me and many others use our vaults. This is a fairly standard feature but I hope it doesn't add too much on your plate.

**Comment by wwjCMP** [comment]
This can actually be achieved now.

**Comment by ghost** [comment]
Could you elaborate on how this can be achieved now @wwjCMP ? 

I am looking for the same functionality.

**Comment by logancyang** [comment]
You can do this with the existing command "set exclusion to vault QA mode". Will need to create a doc site for Vault QA since there are a lot of how-to questions.
